### PR TITLE
Chloe Dygert Crash Analysis Blog Post

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
Maybe incorporate some of Jaap's thoughts:

> It might have been a weave instability, which would explain the low
frequency. For weave, it is quite high.
> Even if the mode is linearly stable, it might be excited by a large
initial disturbance and be unstable in the non-linear regime. Even if you
do not crash, you tend to go approximately in a straight line, so it is
hard to follow the curve.
> In some cases, the cyclist actively trying to stabilize the bicycle makes
things worse.

File: content/dygert-crash.rst [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+For example, on the Single Track Vehicle Dynamics discussion list there are
+`several perspectives`_. As is common with videos showing oscillation of this
+nature, the words "wobble" and "shimmy" are often brought up.
+
+.. _several perspectives: https://groups.google.com/forum/#!msg/stvdy/K932UhSVOAs/uUJE83YuBAAJ
+
+What is "wobble" and "shimmy" of a bicycle?
+===========================================
+
+The words (speed) wobble and shimmy are both interchangeably used in both
```
Comment (yumberry) [inline content/dygert-crash.rst:45]: assuming "The words (speed) wobble and shimmy" are understandable in your world? I'm confused.

File: content/dygert-crash.rst [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+academic and popular literature to describe higher frequency oscillations that
+single track vehicles (bicycles, motorcycles, and similar vehicles) sometimes
+exhibit [Wikipedia2020]_. There does not seem to be a definitive unique
```
Comment (yumberry) [inline content/dygert-crash.rst:48]: [However], there does not seem to be a definitive...

File: content/dygert-crash.rst (hunk 1 above)
Comment (yumberry) [inline content/dygert-crash.rst:45]: I would change to "The words (speed) wobble and shimmy are both interchangeably used in both academic and popular literature without a unique or definitive definition. Both terms are used to describe..."

File: content/dygert-crash.rst [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+oscillation ([Plöchl2012]_, [Klinger2014]_) and, at least in these papers,
+wobble *is* specifically defined. These models show that:
+
+- at speeds > 13 m/s (45 km/h) the wobble frequency of oscillation is between 8
+  and 12 Hz
+- the dominant motion during wobble is the oscillation of the handle bar and
+  fork about the steering axis and this steer motion has an amplitude magnitude
+  of 3-5X that of the roll motion
+- there is a speed threshold (2-6 m/s) at which the oscillation will tend to
+  grow unstably; above this threshold damping this oscillation is the
```
Comment (yumberry) [inline content/dygert-crash.rst:75]: grown unstable?
